test/circle/lint.sh

```diff
@@ -53,15 +53,29 @@ yamllinter() {
   fi
 }
 
+# Validate the Chart.yaml
+validate_chart_yaml() {
+
+  run yamale -s test/circle/yaml-schemas/Chart.yaml ${1}/Chart.yaml
+
+  echo "Validating maintainers names"
+
+  for name in $(yq -r '.maintainers|.[]|.name' ${1}/Chart.yaml); do
+    if [ $(curl -s -o /dev/null -w "%{http_code}\n" -If https://github.com/${name}) -ne 200 ]; then
+      echo "Error: ${name} is not valid GitHub account"
```

Comment:
I like this check. Would it be possible to add some polite language letting people know we have that as a requirement on the charts repo so that we can contact chart maintainers in issues/PRs.